src/components/OnboardingComponents/SemanticSearch/QuerySuggestions.tsx

```diff
@@ -0,0 +1,48 @@
+// SPDX-FileCopyrightText: 2024 German Aerospace Center (DLR)
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import {Box, Chip, Typography} from '@mui/material';
+import {useTranslation} from 'react-i18next';
+
+interface QuerySuggestionsProps {
+  /** A list of suggestion strings to display. */
+  suggestions: string[];
+  /** The function to call when a suggestion chip is clicked. */
+  onSuggestionClick: (query: string) => void;
+}
+
+/**
+ * A component that displays a list of clickable query suggestion chips.
+ */
+export default function QuerySuggestions({suggestions, onSuggestionClick}: QuerySuggestionsProps): JSX.Element {
+  const {t} = useTranslation('global');
+
+  return (
+    <Box mb={2}>
```

Comment:
[System props have been deprecated by MUI](https://mui.com/material-ui/react-box/#with-mui-system-props) and we use the sx props everywhere. Also please use the full keywords:
```suggestion
    <Box
      sx={{
        marginBottom: 2
      }}
    >
```

Reply:
Chaged to:

    <Box
      sx={{
        marginBottom: 2,
      }}
    >
      <Typography
        variant='caption'
        sx={{
          color: 'GrayText',
          marginBottom: 1,
          display: 'block',
        }}
      >